cfme/v2v/migrations.py

```diff
@@ -0,0 +1,329 @@
+                            'mappings': [
+                                {
+                                    'sources':['item1','item2'],
+                                    'target':['item_target']
+                                },
+                                ...
+                            ]
+                        },
+                        ...
+                    }
+        """
+        source_datastores_filled = []
+        target_datastores_filled = []
+        for cluster in values:
+            self.cluster_selector.fill(cluster)
+            for mapping in values[cluster]['mappings']:
+                source_datastores_filled.append(self.source_datastores.fill(mapping['sources'],
+                    partial_match=partial_match))
+                target_datastores_filled.append(self.target_datastores.fill(mapping['target'],
+                    partial_match=partial_match))
+                self.add_mapping.click()
+        was_change = any(source_datastores_filled) and any(target_datastores_filled)
+        if was_change:
+            self.next_btn.click()
+        return was_change
+
+
+class InfraMappingWizardNetworksView(View):
+    include_buttons_set1 = View.include(InfraMappingCommonButtonsSet1)
+    include_buttons_set2 = View.include(InfraMappingCommonButtonsSet2)
+    source_networks = MultiSelectList('source_networks')
+    target_networks = MultiSelectList('target_networks')
+    next_btn = Button("Create")  # overriding, since 'Next' is called 'Create' in this form
+    cluster_selector = BootstrapSelect(id='cluster_select')
+
+    def fill(self, values):
+        """Use to add all mappings specified in values.
+        Args:
+            values:
+                format as following:
+                    {
+                        'cluster_name': {
+                            'mappings': [
+                                {
+                                    'sources':['item1','item2'],
+                                    'target':['item_target']
+                                },
+                                ...
+                            ]
+                        },
+                        ...
+                    }
+        """
+        source_networks_filled = []
+        target_networks_filled = []
+        for cluster in values:
+            self.cluster_selector.fill(cluster)
+            for mapping in values[cluster]['mappings']:
+                source_networks_filled.append(self.source_networks.fill(mapping['sources']))
+                target_networks_filled.append(self.target_networks.fill(mapping['target']))
+                self.add_mapping.click()
+        was_change = any(source_networks_filled) and any(target_networks_filled)
+        if was_change:
+            self.next_btn.click()
+        return was_change
+
+
+class InfraMappingWizardResultsView(View):
+    include_buttons_set1 = View.include(InfraMappingCommonButtonsSet1)
+    next_btn = Button("Close")
```

Comment:
Why `next_btn` and not `close`?

Reply:
because there is no next_btn in this view and including the button_set1, brings next_btn by default, so I override it to point to correct button. If you say, we can have close button pointing to next button and both will be basically pointing to Button("Close").  Thoughts?

Reply:
Sounds like it may be better to not include the CommonButtonsSet1 and just directly instantiate the buttons that are present on this view.